content/en/docs/concepts/storage/volumes.md

```diff
@@ -1316,6 +1316,7 @@ Learn how to
 #### CSI ephemeral volumes
 
 {{< feature-state for_k8s_version="v1.15" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.16" state="beta" >}}
```

Comment:
nit: You could remove line 1318 as the feature is now beta.

Reply:
Same here: I find history important.

Reply:
@pohly  They can always reference ```command-line-tools-reference/feature-gates.md``` for 1.15 state.

Reply:
hello @pohly. The best place to record the state and history of a feature/feature gate is the feature gates reference page. If needed, you could link to the reference page. I _think_ the `feature-state` shortcode is typically used to label the current state of the feature.

Reply:
The k8s.io site has a toggle for release (or 1.14, 1.15, etc) right?

Reply:
Yes

Reply:
Changed.